web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerWps.java

```diff
@@ -0,0 +1,170 @@
+                "  <ows:Identifier>gs:GoGoDuck</ows:Identifier>\n" +
+                "  <wps:DataInputs>\n" +
+                "    <wps:Input>\n" +
+                "      <ows:Identifier>layer</ows:Identifier>\n" +
+                "      <wps:Data>\n" +
+                "        <wps:LiteralData>" + layer + "</wps:LiteralData>\n" +
+                "      </wps:Data>\n" +
+                "    </wps:Input>\n" +
+                "    <wps:Input>\n" +
+                "      <ows:Identifier>subset</ows:Identifier>\n" +
+                "      <wps:Data>\n" +
+                "        <wps:LiteralData>TIME," + literalTime + "," + literalTime + ";LATITUDE,-31.6855,-31.6855;LONGITUDE,114.8291,114.8291</wps:LiteralData>\n" +
+                "      </wps:Data>\n" +
+                "    </wps:Input>\n" +
+                "    <wps:Input>\n" +
+                "      <ows:Identifier>format</ows:Identifier>\n" +
+                "      <wps:Data>\n" +
+                "        <wps:LiteralData>text/csv</wps:LiteralData>\n" +
+                "      </wps:Data>\n" +
+                "    </wps:Input>\n" +
+                "  </wps:DataInputs>\n" +
+                "  <wps:ResponseForm>\n" +
+                "    <wps:RawDataOutput mimeType=\"application/octet-stream\">\n" +
+                "      <ows:Identifier>result</ows:Identifier>\n" +
+                "    </wps:RawDataOutput>\n" +
+                "  </wps:ResponseForm>\n" +
+                "</wps:Execute>";
+    }
+
+    private String getTime() {
+        String time = null;
+        try {
+            HttpURLConnection connection;
+            connection = (HttpURLConnection) (new URL(wfsUrl)).openConnection();
+            connection.setConnectTimeout(LinkMonitorService.timeout * 1000);
+            connection.setReadTimeout(LinkMonitorService.timeout * 1000);
+            connection.setRequestMethod("GET");
+            connection.connect();
+
+            logger.debug(String.format("%s -> %d", wfsUrl, connection.getResponseCode()));
+
+            if (connection.getResponseCode() != 200) {
+                logger.info(String.format("link broken uuid='%s', url='%s', error='bad response code %d'",
+                        this.uuid, this.wfsUrl, connection.getResponseCode()));
+            }
+
+            if (connection.getContentType().equals("text/csv;charset=UTF-8")) {
+                InputStream is = null;
+                CSVReader reader;
+                try {
+                    is = connection.getInputStream();
+                    reader = new CSVReader(new InputStreamReader(connection.getInputStream()));
+                    String[] line;
+
+                    int lineCount = 1;
+                    while ((line = reader.readNext()) != null && line.length == 5) {
+                        if (lineCount == 2) {
+                            time = line[4];
```

Comment:
Column 4? Would it be better to factor this into a variable with a name that explains what's going on.